### ISSUE TITLE
VIPS-WARNING

### ISSUE THREAD

**Issue by auriou** [kleisauke/net-vips#79]
How to remove warning :

(NetVips:1): VIPS-WARNING **: 14:20:34.980: bad exif meta "exif-ifd3-GPSVersionID"

does this cause problems or is it just a warning ?

**Comment by auriou** [comment]
hello, do you know how to define this variable, to disable VIPS warnings, I found this for php

https://github.com/libvips/php-vips-ext/issues/20

My solution is a webapi .net core 3.1

Thanks

**Comment by kleisauke** [comment]
Hi @auriou,

Sorry for the delay. The `bad exif meta "exif-ifd3-GPSVersionID"` is a harmless warning caused by a bug in the EXIF parsing library that libvips uses, see: https://github.com/libvips/libvips/issues/394#issuecomment-359780578.

You can use this to hide all warning messages:
```csharp
// Hide all warnings from vips (similar to the VIPS_WARNING=0 environment variable)
var handlerId = Log.SetLogHandler("VIPS", Enums.LogLevelFlags.Warning, null);

using (var image = Image.NewFromFile("image.jpg", access: Enums.Access.Sequential))
{
    image.WriteToFile("image.png");
}

Log.RemoveLogHandler("VIPS", handlerId);
```

**Comment by auriou** [comment]
hi @kleisauke , great, thank you i will try.

**Comment by kleisauke** [comment]
The `bad exif meta "exif-ifd3-GPSVersionID"` warning will be fixed within libvips 8.10, see the above mentioned issue for details.
